cmd/logger.go

```diff
@@ -17,78 +17,68 @@
 package cmd
 
 import (
+	"encoding/json"
 	"fmt"
-	"io/ioutil"
+	"os"
 	"path"
+	"path/filepath"
 	"runtime"
 	"strings"
+	"time"
 
-	"github.com/Sirupsen/logrus"
 	"github.com/minio/mc/pkg/console"
 	"github.com/minio/minio/pkg/errors"
 )
 
 var log = NewLogger()
 
-// LogTarget - interface for log target.
-type LogTarget interface {
-	Fire(entry *logrus.Entry) error
-	String() string
-}
+// Level type
+type Level int8
 
-// BaseLogTarget - base log target.
-type BaseLogTarget struct {
-	Enable    bool `json:"enable"`
-	formatter logrus.Formatter
-}
+// Enumerated level types
+const (
+	Error Level = iota
+	Fatal
+)
 
-// Logger - higher level logger.
-type Logger struct {
-	logger        *logrus.Logger
-	consoleTarget ConsoleLogger
-	targets       []LogTarget
-	quiet         bool
+func (level Level) String() string {
+	switch level {
```

Comment:
we can have a single return statement here.. 

```go
var lvlStr = "UNKNOWN"
switch level {
case Error: 
      lvlStr = "ERROR"
case Fatal:
      lvlStr = "FATAL"
}
return lvlStr
```

Reply:
Done.